src/cc/CMakeLists.txt

```diff
@@ -142,7 +142,7 @@ if(LIBBPF_FOUND)
   install(TARGETS bcc-shared-no-libbpf LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
 
-install(TARGETS bcc-shared LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(TARGETS bcc-shared bcc-static bcc-loader-static LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
It would be great if it also did this for libbpf.a, see https://github.com/iovisor/bpftrace/blob/master/docker/Dockerfile.alpine#L36

This is where bpftrace currently gets libbpf from, so may as well bundle it too?

Reply:
Sure thing, done.

Reply:
I'm not sure that I did the right thing though, since on my system (debian stretch) it's called `libbcc_bpf.a`, it would be good if you double check.

Reply:
It looks to be the name used in newer versions of bcc, the alpine version is out of date (0.8.0)

https://github.com/iovisor/bpftrace/blob/9c34625d0cfdf447e46cb570af229911485b20af/docker/Dockerfile.bionic-static#L49 references 0.12.0 and uses this new name.

See:

https://github.com/iovisor/bcc/blob/92faebdd7321ebd323b92e1992fd0e7b3eed4c5a/src/cc/CMakeLists.txt#L48 

But we don't reference by the output name here, so I think you need to use `bpf-static` @bobrik